Firmware/lcd.h

```diff
@@ -54,6 +54,7 @@ extern void lcd_printFloat(double number, uint8_t digits);
 
 extern void lcd_print(const char*);
 extern char lcd_print_pad(const char* s, uint8_t len);
+char lcd_print_pad_P(const char* s, uint8_t len);
```

Comment:
why not make this one also extern?

Reply:
I don't see a reason to use `extern` here. We are including the header file in `.cpp` files this function is used so `extern` is not useful here. We could remove `extern` keyword from most of the functions in the firmware without any issues.

Reply:
I agree. I mentioned it just for consistency with the other prototypes around it